catalog/templates/catalog/docs/about.html

```diff
@@ -90,12 +90,19 @@ <h4 id="citation">Citation</h4>
         under the supervision of <a href="https://www.inouyelab.org/home/people">Michael Inouye</a> (<a href="https://sysgenresearch.org/">University of Cambridge & Baker Institute</a>)
         in collaboration with <a href="{{ hdr_uk_cam_url }}">Health Data Research - UK</a> (Laurent Gil)
         and the <a href="{{ ebi_url }}/gwas/">EBI Samples, Phenotypes and Ontologies team / NHGRI-EBI GWAS Catalog</a> (Helen Parkinson, Aoife McMahon, Laura Harris).</p>
-    <p>The Catalog is under active development, and we continue to add new features and curate new data. If you use the Catalog or Calculator in your research we ask that you cite our recent publication:
+    <p>The Catalog is under active development, and we continue to add new features and curate new data. If you use the Catalog or Calculator in your research we ask that you cite our below flagship publications:
     <div class="clearfix mb-3">
       <div class="highlight_content">
```

Comment:
Wondering, as the code is the same as in the FAQ, if we should create a small template for the citation and `include` it for the about and FAQ pages

Reply:
Yes that makes sense, I will add that.